koku/masu/database/sql/reporting_gcpcostentrylineitem_daily_summary.sql

```diff
@@ -0,0 +1,128 @@
+-- TODO:
+-- 1) Figure out what to do with: usage_to_pricing_units
```

Comment:
Do I need to capture the `usage_to_pricing units` in the daily_summary?

Reply:
I think so, or we need to do the unit conversion ourselves. Since they already did it might as well spare the computation. Example: In the API we don't want to show byte-seconds, so if we just use the usage in the pricing unit in the API we're probably good to go.